features/session/session.ts

```
@@ -176,17 +135,71 @@ export class _Session<DataType> {
+  setSessionTimeout(value: number) {
+    globalThis.__SESSION_TIMEOUT__ = value;
+  }
+
+  /**
+   * need to be inited for using session
+   */
+  async initData() {
+    if (!this.request) return;
+    switch (globalThis.serverConfig.session?.type) {
+      case "cookie":
+        const sessionData = this.request.webtoken.session();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)